class.jetpack.php

```diff
@@ -1491,6 +1494,36 @@ public static function is_development_mode() {
 	}
 
 	/**
+	 * Filter enqueued script and minify Jetpack's ones
+	 */
+	public function load_minified_scripts( $tag, $handle, $src ) {
+		$jetpack_url = plugins_url( '', __FILE__ );
+
+		$load_minified_scripts = ! ( ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ) ||
+		                             Jetpack::is_development_mode() || ( defined( 'IS_WPCOM' ) && IS_WPCOM ) );
+
+		/**
+		 * Filters Jetpack's development mode.
+		 *
+		 * @since 4.0.0
+		 *
+		 * @param bool $development_mode Whether to load minified script.
+		 */
+		$load_minified_scripts = apply_filters( 'jetpack_load_minified_scripts', $load_minified_scripts );
+
+		if ( false !== strpos( $src, $jetpack_url ) && $load_minified_scripts ) {
+			$file_name = basename( parse_url( $src, PHP_URL_PATH ) );
```

Comment:
Or on the line below (just for the comparison) to avoid "changing" the filename for case-sensitive OSes.